README.md

```diff
@@ -1,2 +1,276 @@
+
+
+# Getting Started
+
+## Creating a New Module
+
+This module is designed to be a boilerplate for other modules simply by:
+
+1. cloning
+
+1. stripping out any unnecessary content (e.g. no drivers or subsystems)
+
+  - If you don't have any non-sample, non-test projects:
+
+    1. remove the `zephyr/projects/` subdirectory
+    1. In `zephyr/modules.yml` remove the reference to `zephyr/projects`
+
+
+## Creating a Free-Standing Manifest Repository
+
+This example module is also an appropriate template for creating a
+manifest-only repository because, like modules that also contain a
+manifest file, they may contain the following:
+
+- documentation describing the scope and purpose of the workspace
+  configured by the manifest
+
+- configuration settings for CI pipelines
+
+- scripts unique to the breadth of the workspace
+
+To strip this repository down to be a manifest-only repository:
+
+  1. Remove all files and subdirectories referenced by `zephyr/module.yml`
+  1. Remove the `zephyr/include` subdirectory
+  1. Remove `zephyr/module.yml`
+
+
+## Checking Out Code
+
+The Zephyr module ZXA may be cloned and added to the Zephyr build system
+in exactly the same way as any other repository.
+
+Once the Zephyr development environment has been set up (see [Zephyr Getting Started Guide][3]), then follow instructions here to:
+
+- Get the source code with Zephyr module ZXA as the `west`
+  manifest repository
+- Build and run projects
+
+
+## Get Zephyr and Install Python Dependencies
+
+Clone Zephyr module ZXA and its dependencies into a new `west`
+workspace named :file:`zp`. You'll also install additional Python
+dependencies.
+
+1. Get the Zephyr source code:
+
+      ```bash
+      west init \
+        -m https://github.com/zephyrproject-rtos/example-application \
+        --mr main \
+        ~/zxa
+      cd ~/zxa
+      west update
+      ```
+
+1. Export a `Zephyr CMake package`. This allows CMake to
+   automatically load boilerplate code required for building Zephyr
+   applications.
+
+      ```bash
+      west zephyr-export
+      ```
+
+
+## Build and Run the zxa_example_project
+
+Build the `zxa_example_project` with
+`west build`, changing ``<your-board-name>``
+appropriately for your board:
+
+### Ubuntu
+
+1. Build projects directly from the command line
+
+      ```bash
+      cd ~/zxa
+      west build -p auto -b native_posix \
+        example-application/zephyr/projects/zxa_example_project
+      ```
+
+1. Run projects from the command line
+
+      ```bash
+      ./build/example-application/zephyr/projects/zxa_example_project/native_posix/zephyr/zephyr.exe
+      ```
+
+1. Build and run projects using Zephyr's `twister script`
+
+      ```bash
+      zephyr/scripts/twister \
+        -T example-application/zephyr/projects/zxa_example_project
+      ```
+
+1. Build and run the all projects (including projects and tests)
+   with code coverage using Zephyr's `twister script`
+
+      ```bash
```

Comment:
same as above. Also, multiple projects can come later, let's start with a single one for Zephyr 2.6.0

Reply:
In order to verify functionality of Zephyr 2.6.0 build system we need a richly connected example module.  If you also want a simple one now, set up a second project while we're activating the first.